### DISCUSSION TITLE
How can I create or update the record on the page?

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#8991] [Help]
How can I create or update the record on the page? Create if the record does not exist and then update.

**Comment by joshembling** [comment]
I'm a little confused - this is standard Filament behaviour: https://filamentphp.com/docs/3.x/panels/resources/getting-started

**Reply by ArtMin96** [reply]
Thanks. I mean how can I create a page that will work with only one record? In other words, there is nothing after visiting the page for the first time, it becomes an edit page after saving it. It's not a resource page

**Reply by krekas** [reply]
https://laraveldaily.com/post/filament-edit-only-single-record-custom-page
